src/components/Messages/Message/index.js

```diff
@@ -83,7 +91,9 @@ const renderContent = ({
 		),
 	text && (
 		<MessageBubble inverse={me} msgSequence={msgSequence} quoted={quoted} system={system}>
-			<MessageText text={text} system={system} />
+			{showPostChatUrl && <MessageText className = {createClassName(styles, 'surveyText__content')} system={system} text={'Thank you for chatting with us! How did we do today?'} />}
+			{showPostChatUrl && <Button onClick={() => onClickSurvey()} className = {createClassName(styles, 'surveyButton__content')}> Feedback Survey</Button>}
```

Comment:
@ear-dev @bhardwajaditya Instead of `Feedback survey`, should we name this button `Give Feedback`. So that it sounds to the user to take some action?

Reply:
@bhardwajaditya We decided to keep it `Give Feedback` instead of `Feedback survey` in the meeting until Meis finalizes that. Can you please change it? Thanks :)